### PR TITLE
ENT-2614 Clean php extensions directory before install

### PR DESCRIPTION
**Pull request by Lex-2008**

### CODE REVIEW

**Comment by cf-bottom** [comment]
Thanks for submitting a PR! Maybe @craigcomstock can review this?

File: packaging/common/cfengine-hub/preinstall.sh [hunk 1]
```diff
@@ -347,6 +347,10 @@ if [ -d $PREFIX/httpd/htdocs ]; then
   find $PREFIX/httpd/htdocs -depth -type d -exec rmdir {} \;
 fi
 
+if [ -d $PREFIX/httpd/php/lib/php/extensions/no-debug-non-zts-20170718 ]; then
+  rm $PREFIX/httpd/php/lib/php/extensions/no-debug-non-zts-20170718/*
+fi
```
Comment (vpodzime) [inline packaging/common/cfengine-hub/preinstall.sh]: That's a very weird directory name. Any idea where it comes from?

Comment (Lex-2008) [reply packaging/common/cfengine-hub/preinstall.sh]: Not really. The only mention of it that I found was in https://github.com/cfengine/buildscripts/blob/master/packaging/common/cfengine-hub/postinstall.sh#L70

Comment (craigcomstock) [reply packaging/common/cfengine-hub/preinstall.sh]: I believe it is constructed from the bits of info about the extension:

- no-debug
- non-zts (zts is Zen Thread Safety)
- date stamp

https://serverfault.com/questions/399310/php-extensions-no-debug-non-zts-xxxxxxxx

Comment (vpodzime) [reply packaging/common/cfengine-hub/preinstall.sh]: OK, thanks for the info @craigcomstock!

**Pull request merged**